docs/serverless/06-01-function-cr.md

```diff
@@ -75,57 +75,62 @@ spec:
+
+| Reason                           | Type                 | Description                                                                                                               |
+| -------------------------------- | -------------------- | ------------------------------------------------------------------------------------------------------------------------- |
+| `ConfigMapCreated`               | `ConfigurationReady` | A new ConfigMap was created based on the Function CR definition.                                                          |
+| `ConfigMapUpdated`               | `ConfigurationReady` | The existing ConfigMap was updated after changes in the Function CR name, its source code or dependencies.                |
+| `ConfigMapError`                 | `ConfigurationReady` | The ConfigMap could not be created or updated due to an error.                                                            |
+| `JobFailed`                      | `BuildReady`         | The image with the Function's configuration could not be created due to an error.                                         |
+| `JobCreated`                     | `BuildReady`         | The Kubernetes Job resource that builds the Function image was created.                                                   |
+| `JobRunning`                     | `BuildReady`         | The Job is in progress.                                                                                                   |
+| `JobsDeleted`                    | `BuildReady`         | Previous Jobs responsible for building Function images were deleted.                                                      |
+| `JobFinished`                    | `BuildReady`         | The Job was finished and the Function's image was uploaded to the Docker Registry.                                        |
+| `DeploymentCreated`              | `Running`            | A new Deployment referencing the Function's image was created.                                                            |
+| `DeploymentUpdated`              | `Running`            | The existing Deployment was updated after such changes as the Function's image, scaling parameters, variables, or labels. |
+| `DeploymentFailed`               | `Running`            | The Function's Pod crashed or could not start due to an error.                                                            |
+| `DeploymentWaiting`              | `Running`            | The Function was deployed and is waiting for Deployment's readiness.                                                      |
+| `DeploymentReady`                | `Running`            | The Function was deployed and ready to serve.                                                                             |
```

Comment:
```suggestion
| `DeploymentReady`                | `Running`            | The Function was deployed and is ready.                                                                             |
```

Reply:
"ready to serve" - you mean like pasta? ;) If you want to use "serve," you would rather need to explain what it serves like "deployment is ready to serve the traffic". Still, I wouldn't use it, unless you insist ;)